listener/upload_listener.py

```diff
@@ -64,15 +64,22 @@ def process_upload(msg):
                     tmp_file.write(chunk)
             tmp_file.flush()
             parser = ArchiveParser(tmp_file.name)
-            if not parser.parse():
+            try:
+                parser.parse()
+            except Exception: # pylint: disable=broad-except
+                LOGGER.exception("Unable to parse archive: ")
```

Comment:
Let's log the exception-info here, so we know what happened

Reply:
this is what `LOGGER.exception()` does, it appends the exception as single line after this text.

Reply:
Ah ok! Code inspection looks good, and I stood everything up yesterday but did *not* get much further than that to test. Are you OK with me merging this as-is, or should we ask someone to run electrons thru it first?

Reply:
I think it's not needed, it's good enough to test it after it's merged, these are quite straight-forward changes.